test/1.4/registers/instrumentation_test.py

```diff
@@ -42,11 +42,11 @@ def test(obj):
     # the instrumentation tests were disabled
     instrumentation_access_inquire.test(obj, subscribe_b2)
     with stest.allow_log_mgr(None, 'spec-viol'):
-        instrumentation_access_set_missed.test(obj, subscribe_b1, obj)
+        instrumentation_access_set_missed.test(obj, subscribe_b1, obj.bank.b1)
     instrumentation_access_set_offset.test(obj, subscribe_b1)
     instrumentation_access_set_value.test(obj, subscribe_b1)
     instrumentation_access_suppress.test(obj, subscribe_b1)
-    instrumentation_access_value_at_miss.test(obj, subscribe_b1, obj)
+    instrumentation_access_value_at_miss.test(obj, subscribe_b1, obj.bank.b1)
```

Comment:
I think we can remove the last arg, all callers pass obj.bank.b1 now

Reply:
I think so too. I'll try it.

Reply:
yup.